0128-energy-subnetwork.md

```diff
@@ -311,16 +316,17 @@ This shift presents both challenges and opportunities:
+By being able to partner with the existing community of builders it’s a clear win for the whole ecosystem if subnetworks with a strong demand and use case are added. There are already hundreds of thousands Hotspots deployed in the world that can be utilized for the ENERGY subnetwork with a software update. This creates not only an opportunity for quicker growth of the ENERGY subnetwork, but also new incentives for those who are already operating a Helium Hotspot and have renewable energy production in their home. We see this as the natural evolution Helium ethos - building people-powered critical and decentralized infrastructure.
 
-The ENERGY network will also help strengthen the IOT network as the software for reading and controlling DERs is running on the same Hotspot. This means a broader buildout of the IOT coverage to regions beyond highly populated areas with existing coverage. There will also be a need for sensors, such as certified energy meters, sensors for measuring state of the grid such as frequency and voltage and solar lux meters. These sensors will be utilizing the Helium IOT, creating a stronger demand for IOT data. One important aspect is also the anti-gaming perspective: when a new subnetwork is created we can leverage both the ENERGY and IOT network for cross-validation. It is essential that the physical location is correct - and both networks benefit from the other’s validation. For instance, solar panels can be validated both by sensor data (such as energy meters running on Helium IOT) and satellite data where solar panels can be seen from space. As the Helium Hotspot is utilizing the same cryptographic chip for both networks it will be yet another layer of validation added that both networks can leverage from.
+The ENERGY subnetwork will also help strengthen the IoT subnetwork as the software for reading and controlling DERs is running on the same Hotspot. This means a broader buildout of the IoT coverage to regions beyond highly populated areas with existing coverage. There will also be a need for sensors, such as certified energy meters, sensors for measuring state of the grid such as frequency and voltage and solar lux meters. These sensors will be utilizing the Helium IOT, creating a stronger demand for IoT data. One important aspect is also the anti-gaming perspective: when a new subnetwork is created we can leverage both the ENERGY and IOT subnetwork for cross-validation. It is essential that the physical location is correct - and both networks benefit from the other’s validation. For instance, solar panels can be validated both by sensor data (such as energy meters running on Helium IoT) and satellite data where solar panels can be seen from space. As the Helium Hotspot is utilizing the same cryptographic chip for both networks it will be yet another layer of validation added that both networks can leverage from.
```

Comment:
```suggestion
The ENERGY subnetwork will also help strengthen the IOT subnetwork as the software for reading and controlling DERs is running on the same Hotspot. This means a broader buildout of the IOT coverage to regions beyond highly populated areas with existing coverage. There will also be a need for sensors, such as certified energy meters, sensors for measuring state of the grid such as frequency and voltage and solar lux meters. These sensors will be utilizing the Helium IOT, creating a stronger demand for IOT data transfer. One important aspect is also the anti-gaming perspective: when a new subnetwork is created we can leverage both the ENERGY and IOT subnetwork for cross-validation. It is essential that the physical location is correct - and both networks benefit from the other’s validation. For instance, solar panels can be validated both by sensor data (such as energy meters running on Helium IOT) and satellite data where solar panels can be seen from space. As the Helium Hotspot is utilizing the same cryptographic chip for both networks it will be yet another layer of validation added that both networks can leverage from.
```